lms/djangoapps/support/static/support/jsx/single_support_form.jsx

```diff
@@ -149,6 +150,24 @@ class RenderForm extends React.Component {
     return (
       <div className="contact-us-wrapper">
 
+        <StatusAlert
+          alertType="info"
+          open
+          dialog={(
+            <div>
+              <span>Due to the recent increase in interest in online education and edX, we are currently experiencing an unusually high volume of support requests. We appreciate your patience as we work to review each request. Please check the </span>
```

Comment:
Can we internationalize this text? Do we do that on the Contact Us page already?